nextflow.config

```
@@ -283,7 +283,7 @@ manifest {
 
 // Nextflow plugins
 plugins {
-    id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-schema@2.2.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

```suggestion
    id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```
the error has been fixed, so you can stay on this version if you want. (see https://nfcore.slack.com/archives/CE6P95170/p1746789621356379)